### ISSUE TITLE
Unable to serialise []string

### ISSUE THREAD

**Issue by qoke** [gocarina/gocsv#10]
Error generated:

err: No known conversion from []string to string, []string does not implements TypeMarshaller nor Stringernative:
err: No known conversion from []string to string, []string does not implements TypeMarshaller nor Stringerref:

**Comment by pikanezi** [comment]
What would be your expected result to marshalling a slice of strings?

**Comment by abeMedia** [comment]
My suggestion would be adding a configuration option called `SliceSeparator` or similar, with the default being a comma. The cell contents would of course need to be quoted so the commas don't mess up the CSV file. 
Slices of other types could be handled in the same way e.g. `[]int{3,7,12}` => `"3,7,12"`

**Comment by jochavez** [comment]
This functionality would be great to have. Is anyone actively working on it?

**Comment by pikanezi** [comment]
I don't think that anybody is working on it right now, I might do it but in early august as I am busy the two next weeks.

**Comment by theanine** [comment]
+1 to this.  I thought I missed something in the docs before I happened upon this issue.
```
No known conversion from string to []int, []int does not implement TypeUnmarshaller
```
